tests/ui/specs/Scroller/WithFocusOutsideContainer/Scroller-QWT-2652-specs.js

```diff
@@ -0,0 +1,33 @@
+const ScrollerPage = require('../ScrollerPage');
+
+describe('Scroller', function () {
+	describe('5-way LTR', function () {
+		beforeEach(async function () {
+			await ScrollerPage.open('WithFocusOutsideContainer');
+		});
+
+		it('should spotlight an outside of container element with \'native\' scrollMode [QWT-2652]', async function () {
```

Comment:
QWT project also archived.
```suggestion
		it('should spotlight an outside of container element with native scrollMode [QWTC-1998]', async function () {
```